Lib/typing.py

```diff
@@ -208,7 +209,7 @@ def _should_unflatten_callable_args(typ, args):
     """Internal helper for munging collections.abc.Callable's __args__.
 
     The canonical representation for a Callable's __args__ flattens the
-    argument types, see https://bugs.python.org/issue42195. For example:
+    argument types, see https://bugs.python.org/issue42195. For example::
```

Comment:
The double colons are necessary in ReST, but I don't think they make sense in the places where docstrings are normally consumed: they just get displayed as two colons.

Reply:
@sobolevn pointed out that IDEs such as VSCode render the docstrings in a much better way if we use the `::` double colons: https://github.com/python/cpython/pull/105287#discussion_r1217721218